recipes/egl-headers/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, load, save
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class EglHeadersConan(ConanFile):
+    name = "egl-headers"
+    description = "EGL Header files."
+    license = "Apache-2.0"
+    topics = ("egl-headers", "egl")
```

Comment:
No need to duplicate the name in `topics`.

```suggestion
    topics = ("egl")
```